includes/transactions.php

```diff
@@ -0,0 +1,466 @@
+                        print "<td>" . $item['price'] . "</td>";
+                    print "</tr>";
+                }
+                ?>
+                <tr><td colspan='2' class='text-right'>Subtotal</td><td><?php print $result['total_item_price']; ?></td></tr>
+                <tr><td colspan='2' class='text-right'>Shipping</td><td><?php print $result['total_shipping']; ?></td></tr>
+                <tr><td colspan='2' class='text-right'>Tax</td><td><?php print $result['total_tax']; ?></td></tr>
+                <tr><td colspan='2' class='text-right'>Total</td><td><?php print $result['total_order']; ?></td></tr>
+            </table>
+            <div class="row">
+                <div class="col-md-6">
+                <?php
+                
+                print "<h4>Billing</h4>";
+                $billing = $result['_embedded']['fx:billing_addresses'][0];
+                $details = "";
+                $details .= $billing['first_name'] . " " . $billing['last_name'];
+                if ($billing['company'] !== "") $details .= "<br>" . $billing['company'];
+                if ($billing['address1'] !== "") $details .= "<br>" . $billing['address1'];
+                if ($billing['address2'] !== "") $details .= "<br>" . $billing['address2'];
+                $details .= "<br>" . $billing['city'] . " " . $billing['region'] . " " . $billing['customer_postal_code'];
+                $details .= "<br>" . $billing['customer_country'];
+                print "<p>" . $details . "</p>";
+                ?>
+
+                <h4>Payment</h4>
+                <?php
+                $payment = $result['_embedded']['fx:payments'][0];
+                if ($payment['type'] == "plastic") {
+                ?>
+                    <dl>
+                        <dt>Processor Response</dt>
+                        <dd><?php print $payment['processor_response']; ?></dd>
+                        <dt><?php print $payment['cc_type']; ?></dt>
+                        <dd><?php print $payment['cc_number_masked']; ?></dd>
+                        <dt>Expiry</dt>
+                        <dd><?php print $payment['cc_exp_month'] . "/" . $payment['cc_exp_year']; ?></dd>
+                    </dl>
+                <?php
+                } else if ($payment['type'] == "paypal") {
+                ?>
+                    <dl>
+                        <dt>Processor Response</dt>
+                        <dd><?php print $payment['processor_response']; ?></dd>
+                        <dt>PayPal ID</dt>
+                        <dd><?php print $payment['paypal_payer_id']; ?></dd>
+                    </dl>
+                <?php
+                }
+                ?>
+                </div>
+                <div class="col-md-6">
+                <?php
+                foreach ($result['_embedded']['fx:shipments'] as $shipment) {
+                    if (count($result['_embedded']['fx:shipments']) > 1 || $shipment['address_name'] != "Me") {
```

Comment:
It is possible for them to use a language string modification to change this from "Me" to something else which makes this a bit confusing. I'm fine with this for now, just an FYI.